tests/lib/rules/camelcase.js

```diff
@@ -292,12 +302,36 @@ ruleTester.run("camelcase", rule, {
             ]
         },
         {
-            code: "var { category_id: category_id } = query;",
+            code: "var { category_id: category_alias } = query;",
```

Comment:
Should we have a test for the non-shorthand non-renaming case as well, to show that it should be valid? (I think it should be valid, since `object-shorthand` could report the redundancy of the property key/value name.)

Reply:
Sounds good to me.